<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMoKeyboard</class>
 <widget class="QWidget" name="QMoKeyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnKeyCmd">
       <property name="text">
        <string>Cmd</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyHistory">
       <property name="text">
        <string>history</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyDf">
       <property name="text">
        <string>df</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyLs">
       <property name="text">
        <string>ls</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyCtrlC">
       <property name="text">
        <string>^C</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="loLayout1">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnKeyTab">
       <property name="text">
        <string>Tab</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyQ">
       <property name="text">
        <string>Q</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyW">
       <property name="text">
        <string>W</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyE">
       <property name="text">
        <string>E</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyR">
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyT">
       <property name="text">
        <string>T</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyY">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyU">
       <property name="text">
        <string>U</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyI">
       <property name="text">
        <string>I</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyO">
       <property name="text">
        <string>O</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyP">
       <property name="text">
        <string>P</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="loLayout2">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnKeyCaps">
       <property name="text">
        <string>Caps</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyBack">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="woterm.qrc">
         <normaloff>:/woterm/resource/skin/back2.png</normaloff>:/woterm/resource/skin/back2.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyA">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyS">
       <property name="text">
        <string>S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyD">
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyF">
       <property name="text">
        <string>F</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyG">
       <property name="text">
        <string>G</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyH">
       <property name="text">
        <string>H</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyJ">
       <property name="text">
        <string>J</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyK">
       <property name="text">
        <string>K</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyL">
       <property name="text">
        <string>L</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="loLayout3">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnKeyShift">
       <property name="text">
        <string>Shift</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyZ">
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyX">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyC">
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyV">
       <property name="text">
        <string>V</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyB">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyN">
       <property name="text">
        <string>N</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyM">
       <property name="text">
        <string>M</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyDel">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../kxftp/qkxftp.qrc">
         <normaloff>:/kxftp/resource/skin/back.png</normaloff>:/kxftp/resource/skin/back.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="loLayout4">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnKeyCtrl">
       <property name="text">
        <string>Ctrl</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyAlt">
       <property name="text">
        <string>Alt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyLeft">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="woterm.qrc">
         <normaloff>:/woterm/resource/skin/left2.png</normaloff>:/woterm/resource/skin/left2.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyRight">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="woterm.qrc">
         <normaloff>:/woterm/resource/skin/right2.png</normaloff>:/woterm/resource/skin/right2.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeySpace">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="woterm.qrc">
         <normaloff>:/woterm/resource/skin/space.png</normaloff>:/woterm/resource/skin/space.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyUp">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="woterm.qrc">
         <normaloff>:/woterm/resource/skin/up.png</normaloff>:/woterm/resource/skin/up.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyDown">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="woterm.qrc">
         <normaloff>:/woterm/resource/skin/down.png</normaloff>:/woterm/resource/skin/down.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeySymAll">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="woterm.qrc">
         <normaloff>:/woterm/resource/skin/keysym.png</normaloff>:/woterm/resource/skin/keysym.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKeyEnter">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="woterm.qrc">
         <normaloff>:/woterm/resource/skin/keyenter.png</normaloff>:/woterm/resource/skin/keyenter.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../kxftp/qkxftp.qrc"/>
  <include location="woterm.qrc"/>
 </resources>
 <connections/>
</ui>
